readme.txt

```diff
@@ -39,7 +39,7 @@ Our global support team is available to answer questions you may have about WooC
 = Requirements =
 
 * WordPress 5.8 or newer.
-* WooCommerce 6.5 or newer.
+* WooCommerce 6.6 or newer.
```

Comment:
I had [the same question](https://github.com/Automattic/woocommerce-payments/pull/3798#discussion_r807234805) a while ago. It seems like this is for a loose requirement, but I did some quick research and it looks like the readme is not used for parsing anymore as from WP 5.8, which is our minimum required version: https://core.trac.wordpress.org/ticket/48520

cc @htdat

Reply:
> 🤔 Since the [minimum required version is set to be 6.2.2](https://github.com/Automattic/woocommerce-payments/blob/696a1dc7c3cd0e5f5af07fc4695da97796cbc832/changelog/dev-bump-versions-4-6-0#L4), should this be the following instead?

This `Requirements` part will be rendered on https://wordpress.org/plugins/woocommerce-payments/. 

IMO, we should keep L-2 policy here. In our statement, we say that we follow L-2 to encourage merchants using newer Woo core version 6.6+. Though, in the implementation code, we still support back to Woo core 6.2, but this is for legacy purpose as we do not want to suddenly force existing merchants to upgrade their Woo core version. We have public docs for this here https://github.com/Automattic/woocommerce-payments/blob/develop/docs/version-support-policy.md in case you might not know it.

>  it looks like the readme is not used for parsing anymore as from WP 5.8, which is our minimum required version: https://core.trac.wordpress.org/ticket/48520

The header lines are not parsed in WP core: 
https://github.com/Automattic/woocommerce-payments/blob/4c0e6cfaac71b5f11e68a5e689d7fc62d3e6bd8e/readme.txt#L4-L6

but not the specifically mentioned lines here.

Anyway, this is the relevant PR https://github.com/Automattic/woocommerce-payments/pull/3357

Reply:
Another point: though these header lines in `readme.txt` are not parsed in WP core to display in WP Admin > Plugins, or check a plugin info before installing it, I think they are still being used to render for WP.org plugin page https://wordpress.org/plugins/woocommerce-payments/. 

In other words, it's still worth updating versions in both `readme.txt` and `woocommerce-payments.php` files.

Reply:
> ... in the implementation code, we still support back to Woo core 6.2, but this is for legacy purposes, as we do not want to suddenly force existing merchants to upgrade their Woo core version...

This answers my question. Thanks 🙏